RadixWallet/Features/FactorSourceAccess/FactorSourceAccess+View.swift

```diff
@@ -0,0 +1,82 @@
+// MARK: - FactorSourceAccess.View
+
+public extension FactorSourceAccess {
+	struct ViewState: Equatable {
+		let title: String
+		let message: String
+		let externalDevice: String?
+		let retryEnabled: Bool
```

Comment:
we have competing standards, I like prefixing all bools with `is` or `should`, but Gustaf does not. so maybe Ghenadie and or Dan can be tie breakers...

Reply:
I actually prefer the `is` prefix as well. Could be helpful to vote a standard among the team and follow it @GhenadieVP @kugel3 @danvleju-rdx